src/app/features/course/courses-section/courses-section.component.ts

```diff
@@ -4,8 +4,8 @@ import {MatDialog} from '@angular/material/dialog';
 import {Course, DeletedItem} from '../../../core/models/course';
 import {FilterPipe} from '../../../core/pipes/filter.pipe';
 import {OrderByPipe} from '../../../core/pipes/order-by.pipe';
-import {CoursesService} from "../courses.service";
-import {DialogComponent} from "../../../shared/dialog/dialog.component";
+import {CoursesService} from '../courses.service';
+import {DialogComponent} from '../../../shared/dialog/dialog.component';
 
 @Component({
```

Comment:
add OnPush strategy to this component. Would be great to add this strategy to each component in app.

Reply:
done